packages/components/modules/messages/CreateGroup/index.tsx

```
@@ -0,0 +1,356 @@
+  const [commit, isMutationInFlight] = useCreateChatRoomMutation()
+  const { setChatRoom } = useChatRoom()
+
+  const onSubmit = handleSubmit((data: any) => {
+    console.log('create group on submit')
+    const dirtyValues = filterDirtyValues({ values: data, dirtyFields })
+    const { title, participants, image } = data
+    const participantsIds = participants.map((member: any) => member.id)
+    const uploadables: { image?: File | Blob } = {}
+    if ('image' in dirtyValues && image && typeof image !== 'string') {
+      uploadables.image = image
+      delete dirtyValues.image
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)